docs/infrastructure/microk8s-quickstart.md

```diff
@@ -0,0 +1,148 @@
+
+   ```bash
+    microk8s.kubectl wait -n kube-system --for=condition=Ready pod -l k8s-app=kube-dns
+
+    microk8s.kubectl wait -n kube-system --for=condition=Ready pod -l k8s-app=hostpath-provisioner
+
+    microk8s.kubectl wait -n default --for=condition=Ready pod -l name=nginx-ingress-microk8s
+    ```
+
+After runing these commands, we have ensured that dns, http, and nginx ingress are enabled and working properly inside the cluster.
+
+## 5. Installing Codacy
+
+### 1. Helm and Tiller
+
+First we must install `helm` onto the cluster. With `helm`, we can easily install charts and manage the lifecycle of the installed artifacts. One example is the ability to rollback a failed install.
+
+1. __Codacy supports up to `helm` version 2.16.3.__
+   __We currently do not support `helm` v3.__
+2. Follow the instructions on installing the `helm` client on the [Preparing to install Codacy documentation](../../docs/index.md).
+
+3. We must now deploy `tiller` onto the cluster.
+   1. First we must set a `serviceaccount` and a `clusterrolebinding` for `tiller` on the microk8s cluster:
+
+      ```bash
+      microk8s.kubectl create serviceaccount --namespace kube-system tiller
+      microk8s.kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
+      ```
+
+   2. Then we can install `tiller`, which will be installed with the same `HELM_VERSION` that you have specified earlier and we wait for tiller to be ready:
+
+      ```bash
+      helm init --service-account tiller
+      microk8s.kubectl -n kube-system wait --for=condition=Ready pod -l name=tiller
+      ```
+
+### 2. Codacy chart
+
+Please refer to [our chart installation documentation](https://github.com/codacy/chart/blob/master/docs/install.md).
+
+Additionally, any `kubectl` command from [our chart installation](https://github.com/codacy/chart/blob/master/docs/install.md) must be executed as a `microk8s.kubectl` command.
+
+#### 1. Default Values
+
+We provide a base (values file)[##values-microk8s.yaml] that you can use for your microk8s Codacy installation. This file provisions a smaller number of component replicas with limited resources (i.e. CPU and memory). Should you want to use this as a template, please pass your `values-microk8s.yaml` file on (4.) of [our chart installation](https://github.com/codacy/chart/blob/master/docs/install.md). Lastly, it is important to correctly set your `codacy-api.ingress.hosts.host` if you are using this file as a template.
+
+#### 2. External Databases
+
+If you have decided to go with the [default values for the installation](####1.-Default-Values), your microk8s cluster will contain a PostgreSQL database within itself. This is not recommended as the data stored in the database will not be as durable if you store it in an external database.
```

Comment:
We really should not make it easy for people to install using the internal postgres, because it will end up causing them lots of problems

Reply:
I agree. I am removing this paragraph.